facia/app/utils/FaciaPicker.scala

```diff
@@ -5,65 +5,139 @@ import experiments.{ActiveExperiments, DCRFronts}
 import implicits.Requests._
 import model.PressedPage
 import play.api.mvc.RequestHeader
+import views.support.Commercial
 
-class FaciaPicker extends GuLogging {
+object FrontChecks {
 
   // To check which collections are supported by DCR and update this set please check:
   // https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/DecideContainer.tsx
   // and https://github.com/guardian/dotcom-rendering/issues/4720
-  val SUPPORTED_COLLECTIONS: Set[String] =
-    Set("dynamic/fast", "dynamic/slow", "fixed/small/slow-IV", "fixed/large/slow-XIV")
+  private val SUPPORTED_COLLECTIONS: Set[String] =
+    Set(
+      /*
+    "dynamic/slow-mpu",
+      pending https://github.com/guardian/dotcom-rendering/issues/5926 and
+      https://github.com/guardian/dotcom-rendering/issues/5821
+       */
+
+      /*
+    "fixed/small/slow-V-mpu",
+      pending https://github.com/guardian/dotcom-rendering/issues/5926
+       */
+
+      /*
+    "fixed/medium/slow-XII-mpu",
+      pending https://github.com/guardian/dotcom-rendering/issues/5926
+       */
+
+      /*
+    "dynamic/package",
+      pending https://github.com/guardian/dotcom-rendering/issues/5196 and
+      https://github.com/guardian/dotcom-rendering/issues/5267
+       */
+
+      /*
+    "news/most-popular"
+      pending https://github.com/guardian/frontend/issues/25448 and
+      https://github.com/guardian/dotcom-rendering/issues/5902
+       */
+
+      /*
+    "dynamic/fast"
+      pending https://github.com/guardian/dotcom-rendering/issues/5782
+       */
+
+      "dynamic/slow",
+      "fixed/small/slow-I",
+      "fixed/small/slow-III",
+      "fixed/small/slow-IV",
+      "fixed/small/slow-V-third",
+      "fixed/medium/slow-VI",
+      "fixed/large/slow-XIV",
+      "fixed/large/slow-XIV",
+    )
 
-  def dcrSupportsAllCollectionTypes(faciaPage: PressedPage): Boolean = {
+  def allCollectionsAreSupported(faciaPage: PressedPage): Boolean = {
     faciaPage.collections.forall(collection => SUPPORTED_COLLECTIONS.contains(collection.collectionType))
   }
 
-  def getTier(faciaPage: PressedPage, path: String)(implicit request: RequestHeader): RenderType = {
```

Comment:
I know you were  working on this earlier so I'm not  sure whether all changes have been pushed or not, but if you're removing the version of `getTier` that takes `faciaPage: PressedPage, path: String` then I think you'll need to refactor `FaciaController.scala` too (seems to be what's currently causing the build to fail at compilation).

I haven't followed the business logic of this issue closely enough to know whether the FaciaController use of `getTier` _needed_ the path or not; if so, I guess we'd need to keep the old method signature in here too?

Reply:
Thanks Pete, now fixed!